.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-	"dreammaker.extoolsDLL": "byond-extools.dll",
+	"dreammaker.extoolsDLL": "byond-extools.dll"
```

Comment:
What’s with removing this comma?

Reply:
JSON verifier complains about it, because BYOND would, if it tried to load this file. VSCode handles it fine though.